esp/src/eclwatch/ESPWorkunit.js

```diff
@@ -312,7 +312,13 @@ define([
             WsWorkunits.WUUpdate({
                 request: request,
                 load: function (response) {
-                    context.updateData(response.WUUpdateResponse.Workunit);
+                    if (response.Exceptions) {
+                        arrayUtil.forEach(response.Exceptions.Exception, function (item, idx) {
+                            context.updateData(response.Exceptions.Exception[idx].Message);
```

Comment:
What is Message here?  I am not sure this will do anything useful - a simple ```context.updateData(response)``` might be better, but only if you actually want to update something with the exceptions (on the WUDetailsPage maybe?) - but given you have posted to the toaster I am not sure there is any need.